jerry-ext/debugger/debugger-tcp.c

```diff
@@ -167,7 +181,8 @@ jerryx_debugger_tcp_send (jerry_debugger_transport_header_t *header_p, /**< tcp
     }
 #endif /* __linux__ */
 
-    ssize_t sent_bytes = send (tcp_p->tcp_socket, message_p, message_length, 0);
+    const jerryx_socket_void_t *message_to_send_p = (const jerryx_socket_void_t *) message_p;
+    jerryx_socket_ssize_t sent_bytes = send (tcp_p->tcp_socket, message_to_send_p, message_to_send_length, 0);
```

Comment:
This was not what the previous review was about. I've suggested to change the signature of `jerryx_debugger_tcp_send` to have its parameter as `uint8_t *message_p`. Then, `send` can be called as `send (tcp_p->tcp_socket, (const jerry_socket_void_t *) message_p, message_to_send_length, 0);`

Reply:
> This was not what the previous review was about. I've suggested to change the signature of `jerryx_debugger_tcp_send` to have its parameter as `uint8_t *message_p`. Then, `send` can be called as `send (tcp_p->tcp_socket, (const jerry_socket_void_t *) message_p, message_to_send_length, 0);`

Tried but failed, jerryx_debugger_tcp_send parameter can not changed to  `const uint8_t *message_p`.
It's will cause new compiling error, because websocket modify the content of message_p

Reply:
Can you just let this patch go, Or modify it whatever you want, reading the code and compiling the code are totally different things.

Reply:
Use `(jerryx_socket_void_t *) message_p`, din't want modify here anymore